wagtail/images/admin_urls.py

```diff
@@ -1,6 +1,13 @@
 from django.urls import path
 
 from wagtail.images.views import chooser, images, multiple
+from wagtail.admin.views.generic.multiple_upload import DeleteView as BaseDeleteView
+
+
+class DummyDeleteView(BaseDeleteView):
+    def post(self, request, *args, **kwargs):
+        return JsonResponse({"success": True})
```

Comment:
not sure that `JsonResponse` is imported in this file.

Reply:
Definitely not. Also, I am unsure as to the purpose/need for the dummy delete view.
It needs at least some comments, if nothing else.

Reply:
Yeah, `JsonResponse` import was indeed missing, already fixed that.

I used this `DummyDeleteView`, as `delete_action` for case where user wants to delete duplicate image (when custom image model is used).

Reply:
> I used this `DummyDeleteView`, as `delete_action` for case where user wants to delete duplicate image (when custom image model is used).

Given it is not doing anything other than return the operation was successful at all time, i would stipulate this is the wrong approach.

Sorry, but it still doesn't explain why it is needed, and why other methods cannot be used.

Reply:
Well, sorry for not providing that information earlier. 
Anyway - `DummyDeleteView` is used here because in case of using custom image model it cannot be saved to database (because some required fields like `alt` in this example are missing) and therefore there isn't any image model instance that can be deleted, using normal DeleteView (if model instance doesn't exist it cannot be deleted so it would throw error). But at the same time, some kind of DeleteView handling is needed, to populate `delete_action`, so "Delete" button is actually clickable for user in wagtail admin panel.